cluster/manifests/kube-node-ready/service.yaml

```diff
@@ -0,0 +1,19 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: kube-node-ready
+  namespace: kube-system
+  labels:
+    application: kube-node-ready
+    kubernetes.io/cluster-service: "true"
+    kubernetes.io/name: "kube-node-ready"
+  annotations:
+    service.beta.kubernetes.io/external-traffic: OnlyLocal
+spec:
+  type: NodePort
+  ports:
+    - port: 80
+      nodePort: 30797
```

Comment:
we picked this because of the `7979` convention 😄  so this should rather be `30979` or `30079`.